device-types/MikroTik/CCR1009-7G-1C-1S+.yaml

```diff
@@ -0,0 +1,32 @@
+manufacturer: MikroTik
+model: CCR1009-7G-1C-1S+
+slug: ccr1009-7g-1c-1splus
+interfaces:
+  - name: ether1
+    type: 1000base-t
+  - name: ether2
+    type: 1000base-t
+  - name: ether3
+    type: 1000base-t
+  - name: ether4
+    type: 1000base-t
+  - name: ether5
+    type: 1000base-t
+  - name: ether6
+    type: 1000base-t
+  - name: ether7
+    type: 1000base-t
+  - name: sfp-sfpplus1
+    type: 10gbase-x-sfpp
+  - name: combo1
+    type: 1000base-x-sfp
+console-ports:
+  - name: serial0
+    type: de-9
+power-ports:
+  - name: power1
+    type: iec-60320-c14
+    maximum_draw: 20
+  - name: power2
+    type: iec-60320-c14
+    maximum_draw: 20
```

Comment:
Please ensure that each file ends with a newline. (Tests are being updated to enforce this.) This ensures clean output when viewing files on a command prompt.

Reply:
Fixed on 4ef0de5